Content.Shared/_RMC14/Xenonids/Crippling/XenoCripplingStrikeSystem.cs

```diff
@@ -61,13 +65,18 @@ private void OnXenoCripplingStrikeHit(Entity<XenoActiveCripplingStrikeComponent>
             victim.ExpireAt = _timing.CurTime + xeno.Comp.SlowDuration;
             victim.SpeedMultiplier = xeno.Comp.SpeedMultiplier;
 
+            _damageable.TryChangeDamage(entity, xeno.Comp.Damage);
```

Comment:
Won't armor doubly affect this damage? It will apply separately to the 35 and 7, which might bring it below the threshold under which armor resists more of the damage.
You can use GetMeleeDamageEvent instead to modify the damage of the Lurker's attack, though you might need to check that you only modify it for one target.